docs/Installing.asciidoc

```diff
@@ -63,7 +63,21 @@ single-instance container. You can spawn a new shell via:
 podman exec -ti openqa /bin/bash
 ----
 
-From there, you can trigger a new job or clone an existing one, e.g.:
+From there, you can clone the openSUSE test files:
+
+[source,sh]
+----
+git clone https://github.com/os-autoinst/os-autoinst-distri-opensuse.git /var/lib/openqa/tests/opensuse
+----
+
+Afterwards you can clone an existing job, e.g.:
```

Comment:
Did something changed lately? I am pretty sure you do not need to clone the test files before you clone an existing job, because that's what `openqa-bootstrap` does in the Dockerfile.

Reply:
Do you think this is a regression?

Reply:
That's why I am asking if something changed.

Looking at the Dockerfile: https://github.com/os-autoinst/openQA/blob/6b21a3a03898709d67fa5ebde8e114b2aa0f26ee/container/single-instance/Dockerfile#L29

and then at the openqa-bootstrap: https://github.com/os-autoinst/openQA/blob/6b21a3a03898709d67fa5ebde8e114b2aa0f26ee/script/openqa-bootstrap#L143

The code is still there, so it might be a change in default settings or a regression.

Reply:
For cloning tests which have a git url in CASEDIR, it is not necessary, as git will be called automatically by openqa.
For os-autoinst-distri-opensuse tests, they need to be cloned (also the needles by the way), but one can influence that in openqa-bootstrap with passing `skip_suse_tests=` (and `skip_suse_specifics=`) so it will be done automatically:

```
% podman run -e skip_suse_specifics= -e skip_suse_tests= --name openqa --device /dev/kvm \
 -p 1080:80 -p 1443:443 --rm -it registry.opensuse.org/devel/openqa/containers/openqa-single-instance
```

Reply:
Just wanted to point out you can use `-e skip_suse_tests=` to enable that feature in the openqa-bootstrap and we have the auto-clone feature but Tina was faster 😄

Reply:
The point is, you should be able to schedule a test with a one-liner like (where the env variables might not be necessary depending on the settings of the job you are cloning):

```
podman run --device /dev/kvm -p 1080:80 -p 1443:443 --rm -it \
registry.opensuse.org/devel/openqa/containers/openqa-single-instance \
https://openqa.opensuse.org/tests/5109561
```

Reply:
I am not following. So do we want to mention to clone the tests and needles in the documentation, or do we want to avoid it?